src/constants/proptypes.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+export const ButtonTypes = {
+  text: PropTypes.string.isRequired,
+  onClick: PropTypes.func.isRequired,
+};
+
+const SelectShape = {
+  name: PropTypes.string.isRequired,
+  val: PropTypes.string.isRequired,
+};
+
+export const SelectTypes = {
+  list: PropTypes.arrayOf(PropTypes.objectOf(SelectShape)),
+  selected: PropTypes.number.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+export const GoodTypes = {
+  good: PropTypes.string.isRequired,
+};
+
+export const GoodsTypes = {
```

Comment:
this name and the prev one are very similar, it can happen that somebody doesn't notice this and thinks they're the same